backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5671690_sys_gh14215_Delivery_Instruction_Notification_message.sql

```diff
@@ -0,0 +1,9 @@
+-- Value: Event_DeliveryInstructionGenerated
+-- 2023-01-13T13:30:32.339Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545223,0,TO_TIMESTAMP('2023-01-13 15:30:31','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Delivery Instruction Created: {0}.','I',TO_TIMESTAMP('2023-01-13 15:30:31','YYYY-MM-DD HH24:MI:SS'),100,'Event_DeliveryInstructionGenerated')
```

Comment:
please prefix thise AD_Messages with `de.metas.deliveryplanning.`

Reply:
done in separate script